src/components/Options/index.js

```diff
@@ -1,36 +1,36 @@
 import React, {Component} from 'react';
+import PropTypes from 'prop-types';
 import './style.css';
 
 class Options extends Component {
 
-    renderOptions = {
-        'size': this.props.options.map((el, i) => {
+    renderOptions = () => {
+        return this.props.options.map((el, i) => {
             return (
                 <div className="option"
-                     key={el.width + '' + i}
+                     key={el + '' + i}
```

Comment:
`el` is an object, you can either replace it with some property here (like it was before) or just remove it if it doesn't have any unique properties